src/introduction/hardware-overview.md

```diff
@@ -1 +1,21 @@
 # Hardware Overview
+
+The crates under the `esp-rs` organization include support for the [ESP32, ESP32-S, ESP32-C, ESP32-H and ESP32-P Series SoCs][espressif-socs].
```

Comment:
Not sure we should mention the P4 here yet, it's not really supported in most cases.